docker-image/run-nlp-profiler-in-docker.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+MOUNT_VOLUME="${PWD}/../"
+TARGET_FOLDER="/nlp_profiler"
+DOCKER_IMAGE_NAME="nlp_profiler"
+
+echo "~~~ Running nlp_profiler in a docker container"
+echo "Mounted volume: ${MOUNT_VOLUME}"
+
+docker build -t ${DOCKER_IMAGE_NAME} .
+
```

Comment:
It's a good practice to show this line executing with all its parameters, how would you do that in bash?

Also look at one of the examples from the resources shared above